docs/examples/stale-data.py

```diff
@@ -0,0 +1,92 @@
+"""
+Stale Data Periods
+===================
```

Comment:
Underline length

Reply:
Updated in 
commit 839c794a37982d4b12951cbb283d33774f27bb03